AK/Debug.h.in

```diff
@@ -122,10 +122,6 @@
 #    cmakedefine01 JOB_DEBUG
 #endif
 
-#ifndef JPEG2000_DEBUG
-#    cmakedefine01 JPEG2000_DEBUG
-#endif
```

Comment:
(commit message isn't quite true, it's supported by safari!)

Reply:
They have yeeted it for the next version: https://bugs.webkit.org/show_bug.cgi?id=178758